library/packages/src/modules/Package.rb

```diff
@@ -38,31 +38,34 @@
 Yast.import "PackageSystem"
 
 module Yast
+  # This module implements support to query, install and remove packages.
+  #
+  # ## Prefer Package to PackageSystem
+  #
+  # Depending on the mode, this module decides if it should interact with PackageSystem (libzypp) or
+  # PackagesProposal. For instance, if you open a module in the AutoYaST UI, calling to
+  # {CheckAndInstallPackages} does not install the package for real. Instead, it adds the package to
+  # the {PackagesProposal} so it gets exported in the profile. However, when running on other modes
+  # (normal, installation, etc.), it just installs the package.
+  #
+  # ## Overriding default behavior
+  #
+  # There might a scenario where you want to force Package to work with the real packages. For
+  # instance, while reading the configuration during a `clone_system` operation: the mode is still
+  # `autoinst_config` but you are dealing with the underlying system. In those cases, you can force
+  # {Package} to work with {PackageSystem}. {Package} to work with {PackageSystem}.
```

Comment:
Repeated text. BTW you also copypasted the duplication to the description of the PR.